src/app/services/arduino-uploader/protocols/avrdude/index.ts

```
@@ -57,14 +58,22 @@ export default class Avrdude extends BaseProtocol {
             this.port.ondisconnect = resolve;
         });
 
-        const startAvrdude = avrdude.cwrap("startAvrdude", "number", [
-            "string",
-        ]);
-
-        let race = await Promise.race([disconnectPromise, startAvrdude(args)]);
-        // if the winner is the disconnect promise, then the port was disconnected and we should stop the other promise
+        const oldConsoleError = console.error;
+        const workerErrorPromise = new Promise((resolve) => {
+            console.error = () => {
+                resolve({ type: "worker-error" });
+            }
```

Now we will hide any errors after uploading